src/index.tsx

```diff
@@ -575,7 +601,14 @@ export default class ReactToPrint extends React.Component<IReactToPrintProps> {
                 return null;
             }
 
-            const value = { handlePrint: this.handleClick };
+            const value = { 
+                handlePrint: (
+                    event: unknown,
+                    content?: (() => React.ReactInstance | null)
+                ) => {
+                    return this.handleClick(event, content);
+                }
```

Comment:
Looking at this, how is it different from what we had before? Why is `{ handlePrint: this.handleClick }` still not good enough? Were the types having issues?

Reply:
You're right. `{ handlePrint: this.handleClick.bind(this) }` is good enough. I have modified this accordingly